integration-test/spark-common/src/test/java/org/apache/uniffle/test/ShuffleUnregisterWithLocalfileTest.java

```diff
@@ -105,9 +105,8 @@ public Map runTest(SparkSession spark, String fileName) throws Exception {
       map = javaPairRDD.collectAsMap();
       shufflePath = appPath + "/1";
       assertTrue(new File(shufflePath).exists());
-    } else {
-      runCounter++;
```

Comment:
Why do we need this change?

Reply:
When running rssWithNetty, the path with shuffleId=0 has been cleaned up, and an assertion exception will be thrown at this time, `assertTrue(fs.exists(new Path(shufflePath)));`

Reply:
@zuston Could you take a look at this place?

Reply:
I just keep origin code logic.

Reply:
OK.